mysql-wordpress-pd/README.md

```diff
@@ -4,367 +4,195 @@
-as the wordpress sample files) called
-`password.txt` and save your password in it. Make sure to not have a
-trailing newline at the end of the password. The first `tr` command
-will remove the newline if your editor added one. Then, create the
-Secret object.
-
-```shell
-tr --delete '\n' <password.txt >.strippedpassword.txt && mv .strippedpassword.txt password.txt
-kubectl create secret generic mysql-pass --from-file=password.txt
-```
-
-This secret is referenced by the MySQL and WordPress pod configuration
-so that those pods will have access to it. The MySQL pod will set the
-database password, and the WordPress pod will use the password to
-access the database.
+{% capture overview %}
+This tutorial shows you how to deploy a WordPress site and a MySQL database on a Kubernetes cluster in Minikube. Both applications use PersistentVolumes and PersistentVolume Claims to store data. 
+
+A [PersistentVolume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) (PV) is a set amount of storage in a cluster, and a [PeristantVolume Claim](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims) (PVC) is a set ammout of storage in a PV. PVs and PVCs are independent from Pod lifecycles and preserve data through restarting, rescheduling, and even deleting Pods. 
+
+**Wanring:** This tutorial uses Minikube to run a local, single-node Kubernetes cluster. Do not make this a permanent WordPress Deployment.
+{: .warning}
+
+{% endcapture %}
+
+{% capture objectives %}
+* Create a PersistentVolume
+* Create a Secret
+* Deploy MySQL
+* Deploy WordPress
+* Clean up
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+{% include task-tutorial-prereqs.md %} 
+
+Download the following configuration files:
+1. [local-volumes.yaml](https://kubernetes.io/docs/tutorials/stateful-application/local-volumes.yaml)
+2. [mysql-deployment.yaml](https://kubernetes.io//docs/tutorials/stateful-application/mysql-deployment.yaml)
+3. [wordpress-deployment.yaml](https://kubernetes.io/docs/tutorials/stateful-application/wordpress-deployment.yaml)
+
+{% endcapture %}
+
+{% capture lessoncontent %} 
+
+## Create a PersistentVolume
+
+MySQL and Wordpress each use a Persistent Volume to store data. While Kubernetes supports many different [Types of PersistentVolumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#types-of-persistent-volumes), this tutorial covers [hostPath](https://kubernetes.io/docs/concepts/storage/volumes/#hostpath).
+
+**Note:** If you already have a Kubernetes cluster running on Google Container Engine, please follow [this guide](https://cloud.google.com/container-engine/docs/tutorials/persistent-disk).
+{: .note}
+
+### Setting up a hostPath Volume
+
+A `hostPath` volume emulates network-attached storage. 
+
+**Warning:** Only use `hostPaths` for developing and testing. Data does not move between nodes. If a Pod dies, the data is lost even if the Pod restarts on a new node. 
```

Comment:
@cody-clark I'm assuming it would still mount a path from the host, but wouldn't move the data between nodes. Give it a try on the PWK playground or Katacoda's playground with multiple nodes. I think the verbatim from the original sounds better. It should just say "this is because `hostPath`.